Gemfile

```diff
@@ -22,6 +22,7 @@ group :test do
 end
 
 gem 'prometheus-client', '~> 1.0'
-gem 'prometheus_gcstat', git: 'git@github.com:gocardless/prometheus_gcstat_ruby'
-
+source "https://rubygems.pkg.github.com/gocardless" do
+  gem "prometheus_gcstat", "0.1.1"
```

Comment:
Do we need to get a new release of this gem? Not sure how old this is.

Reply:
Nothing got changed in the code after this version https://github.com/gocardless/prometheus_gcstat_ruby/commits/master/ except for github configs

Reply:
Fair. Should we allow any 0.1.x version rather than pin the exact version? That will allow for flexibility as versions upgrade.